templates/web/smidsy/about/faq.html

```diff
@@ -3,95 +3,174 @@
+    <dd>If your report contains potentially sensitive material, such as names or addresses, we suggest that you do not include these until the council has replied to you directly and you are in one-to-one contact.</dd>
+    <dd>If your report is not suitable for publication, you should contact your council directly, via the contact details given on their own website.</dd>
+
+    <dt>I’ve seen an inappropriate report/comment</dt>
+    <dd>There’s a ‘report abuse’ button at the foot of every Collideoscope report. Please click on this and we’ll take a look as soon as we can.</dd>
+    <dd>If the report or comment contravenes our rules, we’ll take it down.</dd>
+</dl>
+
+<h2 name="next">What now</h2>
+
+<dl>
+    <dt>The council never replied; what do I do now?</dt>
+    <dd>Councils may not respond to reports, particularly where no action is required of them. If you’d like a reply and want to chase your council, try searching your email for an auto-response; you may be able to find further contact details there.</dd>
+    <dd>If you still have no joy, you could use another handy mySociety website, <a href="https://www.writetothem.com">WriteToThem</a>, to contact your local councillors.</dd>
+    <dd>You might also like to <a href="https://www.cyclinguk.org/groups-listing">join your local cycling campaigning organisation</a> and take action for better cycling provision in your area.</dd>
+
+    <dt>My report is old and outdated. Can I now delete it?</dt>
+    <dd>We have a policy of retaining all Collideoscope reports because they help create an evidence base of cycling incidents across the country. This data is useful for town planners and researchers who can use it as hard evidence to back up decisions and recommendations for safer road policies.</dd>
+    <dd>As new reports are made, older ones drop off the first page for any given area, and are less likely to be discovered by those visiting the site.</dd>
+
+    <dt>Can you delete my name from the site?</dt>
+    <dd>Yes. Please <a href="/about/privacy">check our Privacy Policy</a> for full details of your rights under the GDPR and information about how to request this.</dd>
+</dl>
+
+<h2 name="accounts">Accounts</h2>
+
+<dl>
+    <dt>Do I need to register with Collideoscope to use it?</dt>
+    <dd>No – you can send a report without registering. When you submit your report, we’ll send you an email with a confirmation link.</dd>
+    <dd>If you use Collideoscope regularly, you can avoid this step by creating an account. Once you’ve signed in, your reports will be sent as soon as you click ‘submit’.</dd>
+    <dd>You’ll also be able to see all previous reports you’ve made on one page.</dd>
+
+    <dt>How do I register?</dt>
```

Comment:
I would ditch this question (at least, until we update the auth page :) ), it is confusing.